token-lending/sdk/src/state/lending_market_metadata.rs

```diff
@@ -0,0 +1,56 @@
+use super::*;
+
+use crate::error::LendingError;
+use bytemuck::checked::try_from_bytes;
+use bytemuck::{Pod, Zeroable};
+use solana_program::program_error::ProgramError;
+use static_assertions::{assert_eq_size, const_assert};
+
+/// market name size
+pub const MARKET_NAME_SIZE: usize = 50;
+
+/// market description size
+pub const MARKET_DESCRIPTION_SIZE: usize = 250;
+
+/// market image url size
+pub const MARKET_IMAGE_URL_SIZE: usize = 250;
+
+/// padding size
+pub const PADDING_SIZE: usize = 200;
+
+/// Lending market state
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[repr(C)]
+pub struct LendingMarketMetadata {
+    /// Market name null padded
+    pub market_name: [u8; MARKET_NAME_SIZE],
+    /// Market description null padded
+    pub market_description: [u8; MARKET_DESCRIPTION_SIZE],
+    /// Market image url
+    pub market_image_url: [u8; MARKET_IMAGE_URL_SIZE],
+    /// Padding
+    pub padding: [u8; PADDING_SIZE],
```

Comment:
i don't even think we need padding tbh. we can just realloc the struct if we need more fields later.

Reply:
idk feel like you get so worried about that

Reply:
there are no financial consequences of messing up this realloc lol